docs/temporal_robustness/direct_model.py

```diff
@@ -0,0 +1,72 @@
+import os
+# export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/home/username/anaconda3/lib/
```

Comment:
See my comments on `continuous_model.py`. Also, why are these files in the docs folder?